oclmath/rounding_mode.cpp

```diff
@@ -130,8 +130,8 @@ RoundingMode set_round(RoundingMode r, Type outType) {
 
   int err = _controlfp_s(&oldRound, 0, 0);  // get rounding mode into oldRound
   if (err) {
-    vlog_error("\t\tERROR: -- cannot get rounding mode in %s:%d\n", __FILE__,
-               __LINE__);
+    // vlog_error("\t\tERROR: -- cannot get rounding mode in %s:%d\n", __FILE__,
+    //            __LINE__);
```

Comment:
To be consistent, we can just remove this code as we do for header includes. What do you think? I suppose `vlog_error` is defined by one of the includes.

Reply:
I went with commenting it out because that was the change that had previously been made to the older version of this file:

https://github.com/KhronosGroup/SYCL-CTS/blob/8623df08565e70e711e48d3a13cc87c1bcfcd518/oclmath/rounding_mode.cpp#L125

But I have no preference here -- if you prefer I remove it, I'm happy to.

Reply:
Done.